airbyte-api/src/main/openapi/config.yaml

```diff
@@ -875,16 +940,46 @@ components:
       properties:
         sourceId:
           $ref: "#/components/schemas/SourceId"
+    SourceCreate:
+      type: object
+      required:
+        - name
+        - defaultDockerRepository
```

Comment:
why is it called default?

Reply:
future-proofing to allow specific connector instances to diverge from the global default version